app/views/locations/libraries/show.html.erb

```diff
@@ -8,6 +8,70 @@
   <%= @library.code %>
 </p>
 
+<p>
+	<dl>
+	  <dt><strong>Floors:</strong></dt>
+	  <dd>
+      <% if @library.floors.any? %>
+      <ul>
+        <% @library.floors.each do |f| %>
+        <li><%= f.label %> <%= image_tag(f.floor_plan_image_url, height: '32') if f.floor_plan_image? %></li>
+        <% end %>
+      </ul>
+      <% else %>
+        No floors specified.
+      <% end %>
+		</dd>
+	</dl>
+</p>
+
+<!--
```

Comment:
Does this commented out block need to stay for possible future use? Otherwise we should take it out.